readme.md

```diff
@@ -5,7 +5,8 @@
   <a href="https://discord.gg/gamGsfg">Discord</a> -
   <a href="https://skiftos.org/">Website</a> -
   <a href="https://github.com/skiftOS/skift/releases">Releases</a> -
-  <a href="manual/readme.md">Manual</a>
+  <a href="manual/readme.md">Manual</a> -
+  <a href="manual/running_vm.md">Run as VM</a>
```

Comment:
There is already a link to this file in the readme, no need for two ;)

Reply:
to which, manual or Run as VM both are below the header and here too

Reply:
The manual link can remain but run as vm need to go